charts/ingress-nginx/values.yaml

```diff
@@ -12,164 +12,159 @@ controller:
   image:
     registry: k8s.gcr.io
     image: ingress-nginx/controller
-    # for backwards compatibility consider setting the full image url via the repository value below
-    # use *either* current default registry/image or repository format or installing chart by providing the values.yaml will fail
-    # repository:
+    ## for backwards compatibility consider setting the full image url via the repository value below
```

Comment:
why change this?

Reply:
The `helm-docs` tool picks up this comment, and tries to put it in the `README.md` file. Since this comment isn't really (I'm guessing) intended to be used as documentation like the others are, I put a double `#` to `helm-docs` ignores it.

Reply:
Got it.